internal/pkg/config/env.go

```diff
@@ -40,7 +40,10 @@ type CustomizeEnv struct {
 }
 
 // IsEmpty returns if CustomizeEnv is an empty struct.
-func (c CustomizeEnv) IsEmpty() bool {
+func (c *CustomizeEnv) IsEmpty() bool {
+	if c == nil {
+		return true
+	}
```

Comment:
nit: can't we do this?
```suggestion
func (c *CustomizeEnv) IsEmpty() bool {
	return c == nil
```

Reply:
not quite, because we want this method to still return `true` when the following happens:
```go
env := &CustomizeEnv{} // not nil
require.True(env.IsEmpty())
```